docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/styles/_homelanding.scss

```diff
@@ -0,0 +1,578 @@
+@import "~Core/styles/environments/percussion";
```

Comment:
Bryan covered the rest of the bases, so I'll just point out that this new file is a duplicate from the Core partials directory. Since it's really a content type (Entry) specific group of rules, it should probably be in the Homelanding.scss rather than a partial at all. We're not doing Home/Landing split page specific types anymore it would seem, so no partial really needed. Any code that is more generic will probably be best to hoist into the core partials eventually when/if it becomes necessary  (There are some outstanding questions about approach on microsites having access to base partials for overrides, so depending, some of this may need to be abstracted innto a partial again to allow for that. But that's just a future consideration, not a comment for this PR.)

Reply:
The only difference in this file is I switched out the `~Core/sprites/svg-sprite` for the local `~Styles/sprites/svg-sprite`.

Reply:
My thought is that there isn't a need for a homelanding partial, what is here in the Common partial should be in Homelanding.scss, until a need to hoist some of it to a Core or Common partial arises.